hbt/selection/lepton.py

```diff
@@ -240,6 +240,12 @@ def tau_selection(
 
     TODO: there is no decay mode selection yet, but this should be revisited!
     """
+    # return empty mask if no tagged taus exists in the chunk
+    if not ak.any(events.Tau.pt):
+        # Convinint definition of empty mask with no substructure of the array (dtype=bool)
```

Comment:
```suggestion
        # Convenient definition of empty mask with no substructure of the array (dtype=bool)
        # adding two different columns is required to avoid having a substructure in the type of the empty mask 
```